content/en/security/application_security/api-inventory/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: API Security Inventory
+kind: documentation
+---
+
+## Overview
+
+API Security Inventory monitors your API traffic to provide visibility into the security posture of your APIs, including:
+
+- **Authentication**: type of authentication used (e.g. Basic Auth, API key, etc.)
+- **Public Exposure**: Whether the API is processing traffic from the internet
+- **Production status**: if the API is running in production environment
+- **Sensitive data flows**: Sensitive data handled by the API and flows between APIs.
+- **Attack Exposure**: if the endpoint is targeted by attacks (powered by [Application Threat Management][2])
+- **Vulnerabilities**:  if the endpoint contains vulnerability (powered by [Application Vulnerability Management][3])
+
+Using the API Security Inventory you can:
+
+- see at a glance your publicly exposed endpoints, which are not requiring authenticated requests
+- see at a glance your publicly exposed endpoints, which are handling sensitive data, such as payment card numbers
+- see which endpoints are at risk, and pivot directly into the [Threat Monitoring and Protection](/security/application_security/threats/) service for further investigation or response
+
+{{< img src="security/application_security/api/welcome.png" alt="API Security Inventory main page">}}
+
+## How to Enable
+
+If your library is configured to use [remote configuration][1], then the following tracer versions
+are compatible without any additional configuration.
```

Comment:
Is there any additional configuration if your library isn't compatible? Is it not supported or just requires some extra steps? I think we could make that more clear to help the user.

Reply:
There are some earlier versions, which are supported but require more complicated configuration.

We weren't sure it's worth putting it here in the docs though, and in case there are some beta partners that are willing to do more complex instructions, we can provide some undocumented instructions to them individually.

At the same time, it would be inaccurate to say "earlier versions are unsupported". 

What about rephrasing as `The following library versions are compatible with API Security Inventory. [Remote Configuration][1]  is required.`